PowerEditor/src/Parameters.cpp

```diff
@@ -791,6 +791,10 @@ winVer NppParameters::getWindowsVersion()
 		_platForm = PF_X86;
 		break;
 
+	case PROCESSOR_ARCHITECTURE_ARM64:

```

Comment:
It seems the symbol `PROCESSOR_ARCHITECTURE_ARM64` is not recognized by MinGW32 :(

Reply:
Only [new version](https://sourceforge.net/p/mingw-w64/mingw-w64/ci/master/tree/mingw-w64-headers/include/winnt.h#l4877) has it. Maybe add it to [gcc-fixes.h](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/master/PowerEditor/gcc/gcc-fixes.h)?

Reply:
Corrected.